pkg/rpm/resolver/templates/rpm-resolution.sh.tpl

```diff
@@ -12,7 +12,7 @@ set -euo pipefail
 #  Arch         - sets the architecture of the rpm packages to pull
```

Comment:
One more thing @josegomezr, please add mention of the new template variable and a brief description here.